x-pack/plugins/maps/public/components/layer_panel/join_editor/resources/join.js

```diff
@@ -6,13 +6,12 @@
 
 import _ from 'lodash';
 import React, { Component } from 'react';
-
 import {
   EuiFlexItem,
   EuiFlexGroup,
   EuiButtonIcon,
 } from '@elastic/eui';
-
+import { i18n } from '@kbn/i18n';
```

Comment:
Forgot to i18n the following
* loadError messege Unable to find Index pattern ${indexPatternId}